external-docs/docs/guides/configuration/code-list-config.md

```diff
@@ -0,0 +1,195 @@
+| `license`          | Link to the license of the data set                                                            | *none*                             |
+| `themes`           | List or a single link of the theme(s) covered by the data                                      | *none*                             |
+| `keywords`         | List or a single string of the keywords(s) covered by the data                                 | *none*                             |
+| `sort`             | Sort by (`label` or `notation`) and sort method (`ascending` or `descending`) of the code list | by (`label`), method (`ascending`) |
+
+#### Using `sort`
+
+The `sort` field allows defining the sort `by` and sort `method` fields of the code list. The sort `by` allows defining the field used by the sorting mechanism for sorting the concepts in a code list. This field can be the concept's `label` or the `notation`. The sort `method` allows defining whether the sorting needs to be done in `ascending` or `descending` order.
+
+*If the `sort` field is defined, the concepts without the `sort_order` (see below) will be sorted according to the sort  `by` and sort `method`. The remaining concepts which has the `sort_order` field defined will be sorted according to the `sort_order` field within each concept.*
+
+### Concepts
+
+The following fields can be defined for each of the concept defined in the concepts section:
+
+| **field name** | **description**                                        | **default value** |
+|----------------|--------------------------------------------------------|-------------------|
+| `label`        | Label of the concept                                   | *none*            |
+| `description`  | Description of the concept                             | *none*            |
+| `notation`     | Notation of the concept                                | *none*            |
+| `sort_order`   | A numeric value defining the sort order of the concept | *none*            |
+| `same_as`      | A link to a concept defined elsewhere                  | *none*            |
+| `children`     | List of child concepts                                 | *none*            |
+
+#### Using `sort_order`
+
+The `sort_order` field allows sorting a code list on per concept basis.
+
+*If both the aforementioned `sort` field and the concept's `sort_order` are defined, csvcubed first sorts the concepts with `sort_order`, and then sorts the rest of the concepts according to the `sort` field. In other words, the concept's `sort_order` field has priority over the code list's `sort` field.*
+
+#### Using `same_as`
+
+The `same_as` field allows using a concept defined elsewhere in the internet (e.g. [E92 concept in Geography Linked Data](http://statistics.data.gov.uk/id/statistical-geography/E92000001)).
+
+### Example
```

Comment:
I've just skimmed this doc for now, but I think you're missing somewhere where you say how you point your qube-config.json's column definition at this separate file.

Reply:
@robons I've updated the Dimension configuration table in qube-config.md file for this. Can. you check and let me know if I need to add more info to this end?

Reply:
I think it'd be good to make it clear in this file how you use it. The qube-config.md file is a bit of a rabbit hole and I don't want user to have to go down it to understand how to define a code list. I think this document should be self-contained enough to let a user define a code list without having to refer to other documents (unless it's to understand basics).